src/level.js

```diff
@@ -27,6 +27,28 @@ async function writeLevel (path, value) {
       Data: {
         type: 'compound',
         value: {
+          // ...oldLevel.value?.Data?.value,
```

Comment:
For now, I see this method `writeLevel` is used only in space-squid on new world creation. But of course it would be better to update these values instead server stop in existing worlds as well. However I don't think want to remove existing fields from level.dat, it would be better to preserve them instead, but I'm not sure how it can be done. Maybe we should read the level first since that's possible that not all data is parsed from readLevel? eg:

```ts
async function writeLevel (path, value) {
  let oldLevel
  try {
    const content = await fs.readFile(path)
    oldLevel = await parseAsync(content)
  } catch { }
// ...
```

Reply:
Examples of such fields that I want to be implemented: `Difficulty`, `DayTime`, `LastPlayed`, spawn coords (and probably `GameType`)